_data/blogs.yml

```diff
@@ -222,3 +222,8 @@
   title: Flying with Fossasia
   img: arjun.jpg
   author_name: Arjun Pandey
+  
+- url: https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title: Here I come Coding In
+  img:nil
```

Comment:
You need to write the filename of the image you are uploading here, and make sure to leave a space after `:`, so that the travis CI doesn't fail.

Also, add the file in the `img/blog/`folder.